README.md

```diff
@@ -1,3 +1,25 @@
 # Blackhole.service for DEIS
 
-This will be a repository for dockerized blackhole service for DEIS.
+### Blackhole.service
+
+An example file for fleetctl unit.
+
+```
+  [Unit]
+  Description=Turn on automated addition of blackhole routes for sshd attackers
+
+  [Service]
+  TimeoutStartSec=0
+  ExecStartPre=-/usr/bin/docker kill deis-blackhole
+  ExecStartPre=-/usr/bin/docker rm deis-blackhole
+  ExecStartPre=/usr/bin/docker pull khelben/deis-blackhole
+  ExecStop=/usr/bin/docker kill deis-blackhole
+  ExecStart=/bin/sh -c '/usr/bin/docker run --name deis-blackhole --net=host --privileged -e WHITELIST="$(fleetctl list-machines --fields=ip -no-legend)" -v /var/log/journal:/var/log/journal:ro -v /usr/bin/journalctl:/usr/bin/journalctl:ro -v /lib64:/usr/local/lib64:ro khelben/deis-blackhole'
```

Comment:
Because this is just a repo for a container, that need to to be build and push to docker hub.